app/models/concerns/deliverable.rb

```diff
@@ -7,11 +7,19 @@ module Deliverable
   end
 
   def deliver_later
-    delivery.deliver_later
+    job = delivery.deliver_later
+
+    if respond_to?(:record)
+      record(:delivery_enqueued, sidekiq_job_id: job.provider_job_id)
+    end
   end
 
   def deliver_now
     delivery.deliver_now
+
+    if respond_to?(:record)
+      record(:sent, recipient_email_address: delivery.to)
+    end
```

Comment:
When we call `deliver_later`, are we planning on recording a `:sent` when the email actually sends? If so, we should probably abstract this into its own method so that it can be called from the job (such as `digest.record_delivery_sent`)

Reply:
Yeah, I was trying to tiptoe around potential abstractions. I'll just not record anything for now and I can figure out a good system in a different PR.